test/JDBC/README.md

```diff
@@ -415,3 +416,29 @@ After building the modified PostgreSQL engine and Babelfish extensions using the
     unset isParallelQueryMode
    ```
 If you encounter failing or crashing tests in the "JDBC tests with parallel query" GitHub workflow, consider adding the names of these problematic test cases to the `parallel_query_jdbc_schedule` file. Prefix these test case names with `ignore#!#`. As we work towards resolving these issues in the future, we will gradually remove these excluded tests from the `parallel_query_jdbc_schedule` scheduling file.
+
+## Running Tests with Non Default Server Collation
+
+After building the modified PostgreSQL engine and Babelfish extensions using the [online instructions](../../contrib/README.md), you must:
+1. Create a PostgreSQL database and to initializing Babelfish extensions with non-default server collation add following line with server collation name of your choice in `postgres/data/postgresql.conf` and restart engine, then initialize Babelfish extensions using the [online instructions](../../contrib/README.md)
+
+   ```bash
+   babelfishpg_tsql.server_collation_name = '<server_collation_name>'
+    ```
+2. Before running JDBC tests, set the `serverCollationName` environment variable to the current server collation name:
+
+   ```bash
+    export serverCollationName=<server_collation_name>
+    # Verify if serverCollationName is set to correct collation name
+    echo $serverCollationName
+   ```
+3. Now Run the tests:
+    ```bash
+    mvn test
+    ```
+4. If the expected output is different when run with non-default server collation, one can add a different expected output in `expected/non_default_server_collation/<server_collation_name>` folder. Similary, if tests are running in parallel query mode and expected output is different with non-default server collation, one can add a different expected output in `expected/parallel_query/non_default_server_collation/<server_collation_name>/` folder.
```

Comment:
This seems a bit confusing. If the tests are run in parallel query mode, there might be two cases right? 

1. Difference in output due to parallel query mode and non default server collation, in this case we add the expected output in `expected/parallel_query/non_default_server_collation/<server_collation_name>/`
2. Difference in output only due to non default server collation, we add it in `expected/non_default_server_collation/<server_collation_name>`

The above point does not represent this. Can you please update it ?

Reply:
If the tests are run in parallel query mode, two cases are as follows
case 1: parallel query enabled with default server collation
- adding different expected output in this case remains same [Reference](https://github.com/babelfish-for-postgresql/babelfish_extensions/tree/BABEL_3_X_DEV/test/JDBC#running-tests-with-parallel-query-enabled).

case 2: parallel query enabled with non-default server collation
- adding different expected output in this case are added in the readme

Anyways I'll add all cases explicitly

Reply:
Done